debian/control

```diff
@@ -19,31 +18,16 @@ Build-Depends: debhelper (>= 13.0.0),
                qtmultimedia5-dev,
                qttools5-dev,
                qttools5-dev-tools,
-               zlib1g-dev,
+               libquazip1-qt5-dev,
 Homepage: https://mediaelch.github.io/mediaelch-doc/
+Rules-Requires-Root: no
 Vcs-Browser: https://github.com/Komet/MediaElch/
 Vcs-Git: https://github.com/Komet/MediaElch.git
 
 Package: mediaelch
 Architecture: any
-Depends: ${shlibs:Depends},
-         ${misc:Depends},
-         libcurl3 | libcurl4,
-         libmediainfo0 | libmediainfo0v5,
-         libpulse0
-         libqt5concurrent5,
-         libqt5core5 | libqt5core5a,
-         libqt5gui5,
-         libqt5multimedia5,
-         libqt5multimediawidgets5,
-         libqt5network5,
-         libqt5opengl5,
-         libqt5sql5,
-         libqt5sql5-sqlite,
-         libqt5svg5,
-         libqt5xml5,
-         libzen0 | libzen0v5,
-         zlib1g,
+Depends: ${misc:Depends},
+         ${shlibs:Depends},
```

Comment:
Could you bring me up to speed here? Why are the runtime dependencies not listed here? At least libmediainfo is loaded at runtime and auto-detection (if any used) fails. That's at least what's happening with openSUSE: https://github.com/Komet/MediaElch/blob/ad330574c22a421524312655b2c52acf86f6509e/obs/MediaElch.spec#L12

Reply:
Oh my god, that is awesome. I wasn't aware of that! I read that page once, but apparently didn't read it properly 😄